tempesta_fw/str.c

```diff
@@ -410,8 +410,48 @@ tfw_str_eq_cstr_pos(const TfwStr *str, const char *pos, const char *cstr,
 }
 EXPORT_SYMBOL(tfw_str_eq_cstr_pos);
 
+bool
+tfw_str_eq_cstr_off(const TfwStr *str, ssize_t offset, const char *cstr,
```

Comment:
Please add header comment for the function and appropriate set of tests in t/unit/test_tfw_str.c